src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -52,18 +55,28 @@ export const useGetTradeQuotes = () => {
           allowMultiHop: flags.MultiHopTrades,
         })
 
-        setTradeQuoteInput(tradeQuoteInputArgs ?? skipToken)
+        // if the quote input args changed, reset the selected swapper and update the trade quote args
+        if (!isEqual(tradeQuoteInput, updatedTradeQuoteInput ?? skipToken)) {
```

Comment:
I don't think we want the right side of the nullish coalescing here? We definitely want to keep it in `setTradeQuoteInput()` call but I don't think we want it as equality check
<img width="1396" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3b9e7b98-3954-4738-94fc-e8b28204ce27">

```suggestion
        if (!isEqual(tradeQuoteInput, updatedTradeQuoteInput ?? skipToken)) {
```

Reply:
we definitely do want to include the skipToken because we want to prevent rerender and only execute it when the input has changed. tradeQuoteInput is `TradeQuoteInput | typeof skipToken`, rather than `TradeQuoteInput | undefined`